### PR TITLE
Rely on OPA extension rather than re-declare the rego language

### PR DESCRIPTION
**Pull request by lkwdwrd**
### What code changed, and why?

Rather than act as an alternative to the existing OPA extension, Styra can build on top of the OPA extension, allowing us to make use of things such as the rego language definition, the OPA binary location, root paths, and more. The goal is to make the experience of authoring rego with VS Code as cohesive as possible.

### Definition of done

When a user installs the VS Code Styra extension, the OPA extension is also installed if they do not have it, providing Rego language support and more.

### How to test

First uninstall both the Styra and OPA extensions if you have them installed in VS Code. Next, re-install the development version of the VS Code Styra extension from GitHub.

1. Go to the latest release on the [release page](https://github.com/StyraInc/vscode-styra/releases).
2. Download the additional version piggybacked into that release, labelled `vscode-styra-n.n.n-next.N.vsix`.
3. Install via the standard command: `code --install-extension vscode-styra-n.n.n-next.N.vsix`

Validate that after installation you have both the OPA extension _and_ the Styra extension installed. The OPA extension will be installed automatically when installing the Styra extension.

### CODE REVIEW

**Comment by srenatus** [comment]
> Download the additional version piggybacked into that release, labelled vscode-styra-n.n.n-next.N.vsix.

Took me moment to spot it 😅 

Untitled

**Review by srenatus — APPROVED** [review]
> Validate that after installation you have both the OPA extension and the Styra extension installed. The OPA extension will be installed automatically when installing the Styra extension.

✅

File: .vscode/settings.json
Comment (chendrix) [inline .vscode/settings.json:1]: Was this change meant to be here?

Comment (chendrix) [reply .vscode/settings.json:1]: Ah I see, it's because the OPA extension is under `tsandall` not like the open policy agent

**Review by chendrix — APPROVED** [review]

**Pull request merged**